### PR TITLE
OH2-294 | Types / Exams CRUD

### PR DESCRIPTION
**Pull request by loique70**
See OH2-294.

### CODE REVIEW

**Review by SilverD3 — CHANGES_REQUESTED** [review]
The function looks good, I mean we can list/edit/add/delete exam types.
But I think you've duplicated the slice `src\state\examTypes`. I suggest to remove it and replace all references by the new state slice `src\state\types\exams`

File: src/components/accessories/admin/types/components/exams/ExamsTypes.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      <h3>{t("examTypes.title")}</h3>
+
+      <div className="examTypes">
+        <ExamTypesTable
+          onEdit={handleEdit}
+          onDelete={handleDelete}
+          headerActions={
+            <Button
+              onClick={() => {
+                navigate("./new");
```
Comment (SteveGT96) [inline src/components/accessories/admin/types/components/exams/ExamsTypes.tsx]:
```suggestion
                navigate(PATHS.admin_exams_types_new);
```

**Review by SteveGT96 — CHANGES_REQUESTED** [review]

**Comment by SilverD3** [comment]
Please fix conflits and take into consideration my [suggestion](https://github.com/informatici/openhospital-ui/pull/608#pullrequestreview-2112966554)

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**